kili/queries/asset/__init__.py

```diff
@@ -236,9 +236,8 @@ def _query_assets(self,
     @Compatible(['v1', 'v2'])
     @typechecked
     @deprecate(removed_in="2.115")
```

Comment:
```suggestion
    @deprecate(removed_in="2.117")
```

Reply:
This has to be changed everywhere. otherwise the bump to 2.115 will be blocked.
We want to block it in the 2.117 if we forgot to remove the deprecation warning

Reply:
done